### ISSUE TITLE
Two finger swipe to pan

### ISSUE THREAD

**Issue by ChaseOnTheWeb** [mpetroff/pannellum#910]
If you have a pano within a page of other content, and that pano is full-width and as tall as the viewport, it makes it tricky to scroll past it because Pannellum wants to pan.

Would it be feasible to add an option requiring two fingers to pan (Google Maps does this), and letting the browser handle one finger touch for scrolling? It looks like you can already pan or zoom with two fingers.

**Comment by strarsis** [comment]
Yes, this would be a useful feature.

**Comment by mpetroff** [comment]
Yes, this would be a good addition but only with a "use two fingers to move" notice (similar to what Google Maps does) to make the behavior discoverable. This is similar to the suggestion in https://github.com/mpetroff/pannellum/issues/360#issuecomment-323026648 to do the same for the mouse wheel.

**Comment by strarsis** [comment]
@mpetroff: PR ready: https://github.com/mpetroff/pannellum/pull/952
